libr/cons/cons.c

```diff
@@ -1322,3 +1322,50 @@ R_API void r_cons_breakword(const char *s) {
 		I.break_word_len = 0;
 	}
 }
+
+/* Prints a coloured help message.
+ * help should be an array of the following form:
+ * {"command", "args", "description",
+ * "command2", "args2", "description"}; */
+R_API void r_cons_cmd_help(const char *help[], bool use_color) {
+	int i, max_length, padding_length;
+	RCons *cons = r_cons_singleton ();
+	char const *args_color_start;
+	char const *help_color_start;
+	char const *reset_colors;
+	char padding[256];
+
+	args_color_start = use_color ? cons->pal.args: "";
```

Comment:
put that assignment in the same definition line or put all 3 assignments in a if (use_color) { block

Reply:
better use different varnames for storing the colors. like "pal_" + ...